src/config/config.go

```diff
@@ -48,6 +49,7 @@ func ProcessArgs() {
 	ret := &GlobalConfig
 	flag.StringVar(&ret.Role, "role", string(models.HostRoleMaster), "The node role")
 	flag.StringVar(&ret.ClusterID, "cluster-id", "", "The cluster id")
```

Comment:
do we still need the 'ClusterID' ?

Reply:
yes, it is used for access the cluster and getting its hosts by assisted-controller